server/player.lua

```diff
@@ -542,6 +555,7 @@ function CreatePlayer(playerData, Offline)
         if not meta or type(meta) ~= 'string' then return end
         if meta == 'hunger' or meta == 'thirst' then
             val = val > 100 and 100 or val
+            Player(self.PlayerData.source).state:set(meta, val, true)
```

Comment:
I know this was not really part of the original pr but this 
```lua
val = val > 100 and 100 or val
```
Is extremely messy and we have (thanks to ox_lib) a better math library therefor please use lib.math.clamp here.

```lua
Player(self.PlayerData.source).state:set(meta, math.clamp(val, 0, 100), true)
```